packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -115,7 +114,7 @@ class Chat extends Widget<Properties> {
   }
 
   _renderMessageList(): void {
-    const { items = [], user, showDayHeaders } = this.option();
+    const { items = [], user, showDayHeaders = true } = this.option();
```

Comment:
Why do we need this change? Looks excess

Reply:
It requires to avoid typescript issue, because showDayHeaders can be 'undefined' according to new types.